src/service/email/EmailService.ts

```diff
@@ -0,0 +1,31 @@
+const nodemailer = require('nodemailer');
+
+export const EmailService = () => {
+  const send = async (from, subject, email, content) => {
+    const { NODEMAILER_EMAIL, NODEMAILER_PASSWORD } = process.env
+
+    let transport = nodemailer.createTransport({
+      service: 'gmail',
+      auth: {
+        user: NODEMAILER_EMAIL,
+        pass: NODEMAILER_PASSWORD
+      }
+    });
+
+    const message = {
+      from: from,
+      to: email,
+      subject: subject,
+      text: content
+    }
+
+    try {
+      const info = await transport.sendMail(message)
+      console.log('Email sent:', JSON.stringify(info))
+    } catch (error) {
+      console.error('Failed to send email:', error.message)
+    }
+
+  }
+  return { send }
```

Comment:
A maneira mais simplificada de implementar isso seria externalizar o `nodemailer` em seu próprio módulo e importá-lo aqui:

```suggestion
import {transporter} from './email-transporter'

export const EmailService = () => {
    const message = {
      from: from,
      to: email,
      subject: subject,
      text: content
    }

    try {
      const info = await transporter.sendMail(message)
      console.log('Email sent:', JSON.stringify(info))
    } catch (error) {
      console.error('Failed to send email:', error.message)
    }

  }
  return { send }
```

Uma vantagem aqui seria que `EmailService` fica um pouco mais desacoplado da biblioteca `nodemailer`, ou seja, se um dia vocês precisarem trocar a implementação de envio de emails, o código de `EmailService` não sofreria tanto impacto, bastaria trocar o import.

Enfim, não é uma mudança que precisa ser feita, é só pra levantar a bola e deixar vocês pensando sobre acoplamento, responsabilidades, etc.